lib/Models/Segment.php

```diff
@@ -166,6 +167,24 @@ public static function getWooCommerceSegment() {
     return $wcSegment;
   }
 
+  public static function getWooMembershipsSegment() {
```

Comment:
Do we need the changes on this file? They don't seem to be used.

Reply:
Good catch! This is a leftover, I removed it.